css-flexbox_profile/css/styles.css

```diff
@@ -0,0 +1,63 @@
+:root {
+    --primary-dark: #2E016D;
+    --primary: #3F0097;
+    --primary-light: #8236FF;
+    --secondary-dark: #38CCDA;
+    --secondary: #7fD9E2;
+    --accent: #FFD15D;
+
+    --black-light: #303030;
+    --black: #000000;
+    --white: #FFFFFF;
+    --grey-dark: #B6B9C2;
+    --grey-light: #F5F6F6;
+}
+
+* {
+  box-sizing: border-box;
+  align-self: center;
+}
+
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  padding: 40px;
+  margin: 0 auto;
+  max-width: 1200px;
+}
+
+.main__title {
+  padding: 10px;
+  background: var(--primary);
+  color: var(--white);
+
+}
+
+.profile {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  gap: 10px;
```

Comment:
I think the `flex-wrap: wrap` is not necessary. Without it looks the same.
10 px gap was not meant to be here.